store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedOperationHandler.java

```diff
@@ -61,6 +62,7 @@ private Iterable getAllGraphResults(final FederatedOperation<INPUT, OUTPUT> oper
         try {
             List<Object> results;
             final Collection<GraphSerialisable> graphs = getGraphs(operation, context, store);
+            context.setVariable(MERGE_FUNCTION_GET_GRAPHS, graphs.stream().map(GraphSerialisable::getGraphId).collect(Collectors.toList()));
```

Comment:
I am a little confused by this change

Reply:
I'm retaining this info into context, so I can smuggle the graphs info for a more meaningful error message if required later in the flow.

Reply:
Ah yeah. Originally this exception was getting thrown else where, but it's been pulled up and up so. So now that it is actually back in the same class, so I probably could just be set to a field variable now :D

Reply:
``` java
    @JsonIgnore
    private List<String> errorMessageGraphIds; // large scope and exclusion from JSON just to improve an error message. 
```

Reply:
Spoken about and resolved